platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functionnal/SOCNotificationIntranetMentionTestIT.java

```diff
@@ -0,0 +1,279 @@
+
+        ArrayList<String> arrayUser = new ArrayList<String>();
+        String username1 = "usernamea" + getRandomString();
+        String email1 = username1 + "@gmail.com";
+        String username2 = getRandomString();
+        String email2 = username2 + "@gmail.com";
+        String password = "123456";
+        arrayUser.add(username1 + " " + username1);
+
+        info("Add new user");
+        navigationToolbar.goToAddUser();
+        userAddManagement.addUser(username1, "123456", email1, username1, username1);
+        userAddManagement.addUser(username2, "123456", email2, username2, username2);
+        manageLogInOut.signIn(username1, "123456");
+        homePagePlatform.goToHomePage();
+        String activity1 = "activitya" + getRandomNumber();
+        activityStream.mentionUserActivity(username2, activity1);
+
+		/*Step Number: 1
+		 *Step Name:
+		 *Step Description:
+			- Login with User B
+			- Click the notification icons in the top navigation
+			- Check the notification list
+		 *Input Data:
+
+		 *Expected Outcome:
+			- The Mention notification is displayed in the list*/
+        manageLogInOut.signIn(username2, password);
+        navigationToolbar.goToIntranetNotification();
+
+		/*Step number: 2
+		 *Step Name:
+		 *Step Description:
+			- Check the notification message
+		 *Input Data:
+
+		 *Expected Outcome:
+			The notification message is : $AVATAR$USER has mentioned you $ACTIVITY$DATEWhere :
+			- $AVATAR is the thumbnail of User A
+			- $USER is User A
+			- $ACTIVITY is the activity title/message
+			- $DATE is the date of the notification*/
+        arrayUser.add(username1);
+        String status = "UserA has mentionned you";
+        intranetNotification.checkAvatarInStatus(username1 + " " + username1, true);
+        intranetNotification.checkStatus(status, username1);
+        intranetNotification.checkActivityTitleInStatus(activity1, true);
+        intranetNotification.checkUsers(arrayUser, true);
+
+        info("Test 4: Click the Mention notifications (in activity message)");
+		/*Step Number: 1
+		 *Step Name:
+		 *Step Description:
+			- Login with User B
+			- Click the notification icons in the top navigation
+			- Check the notification list
+		 *Input Data:
+
+		 *Expected Outcome:
+			- The Mention notification is displayed in the list*/
+
+		/*Step number: 2
+		 *Step Name:
+		 *Step Description:
+			- Click the notification
+		 *Input Data:
+
+		 *Expected Outcome:
+			- The user is redirected to the activity viewer with all comment expanded.*/
+        intranetNotification.checkAvatarInStatus(arrayUser, true);
+        intranetNotification.checkStatus(status, username1);
+        intranetNotification.checkActivityTitleInStatus(activity1, true);
+        refresh();
+        $(byXpath(ELEMENT_ACTIVITY_MENTION_USER.replace("${content}", activity1).replace("${user}", username2))).waitUntil(Condition.visible, Configuration.timeout);
```

Comment:
We must delete the user at the end of the test

Reply:
done